eQuiz/modules/eQuiz.Web/Areas/Admin/Views/web.config

```diff
@@ -29,7 +29,7 @@
   <system.webServer>
     <handlers>
       <remove name="BlockViewHandler"/>
-      <add name="BlockViewHandler" path="*" verb="*" preCondition="integratedMode" type="System.Web.HttpNotFoundHandler" />
+      <add name="BlockViewHandler" path="*cshtml" verb="*" preCondition="integratedMode" type="System.Web.HttpNotFoundHandler" />
```

Comment:
function LayoutController($scope, $http, $sce) {
        var vm = this;
        function GoToReview() {
            $http.get('/Areas/Admin/Views/Pages/Review.html')
            .then(function (response) {
                vm.review = $sce.trustAsHtml(response.data);
            });
        };